tests/acceptance/features/apiCollaboration/wopi.feature

```diff
@@ -493,3 +493,32 @@ Feature: collaboration (wopi)
       | /app/open-with-web?file_id=<<FILEID>>&app_name=FakeOffice&view_mode=view  |
       | /app/open-with-web?file_id=<<FILEID>>&app_name=FakeOffice&view_mode=read  |
       | /app/open-with-web?file_id=<<FILEID>>&app_name=FakeOffice&view_mode=write |
+
+  @issue-9928
+  Scenario Outline: open unsupported file format (open-with-web)
```

Comment:
can we expand this tests for other unsupported file format too

Reply:
no need. one scenario should be fine for now